contract/src/platform-goals/start-governed-contract.js

```diff
@@ -55,12 +54,14 @@ export const ParamTypes = /** @type {const} */ ({
  *   governedContractInstallation: ERef<Installation<SF>>;
  *   issuerKeywordRecord?: IssuerKeywordRecord;
  *   terms: Record<string, unknown>;
- *   privateArgs: StartParams<SF>['privateArgs'];
+ *   privateArgs?: StartParams<SF> extends { privateArgs: unknown }
```

Comment:
What was the problem here? `StartParams` should figure this out itself.

Reply:
`Type '"privateArgs"' cannot be used to index type 'StartParams<SF>'` if its kept as is. Couldn't think of a way out of it - any suggestions?

Reply:
not off hand. please put an `XXX` comment to indicate this is some tech debt to clean up later.

https://github.com/Agoric/agoric-sdk/wiki/Coding-Style#code-comments